### PR TITLE
finished homework

### PR DESCRIPTION
**Pull request by Serafettin99**

### CODE REVIEW

File: homework/index.js [hunk 1]
```diff
@@ -1,54 +1,133 @@
...
+  const root = document.getElementById('root');
+  const header = createAndAppend('header', root);
+  const p = createAndAppend('p', header, {
+    text: 'HYF Repositories',
+  });
+  const select = createAndAppend('select', header);
+  const main = createAndAppend('main', root, {
+    class: 'main-container',
+  });
+  const section1 = createAndAppend('section', main, {
```
Comment (ArcoMul) [inline homework/index.js]: I imagine there can be more descriptive names instead of `section1`, `section2`, `ul`, `ul2`. It's a bit hard to follow now what is where.

Comment (Serafettin99) [reply homework/index.js]: I fixed them as `contributorSection`, `repoSection` etc.

**Comment by Serafettin99** [comment]
Hi, @ArcoMul . Thanks for review and feedback. I struggled a lot while doing homework and spend too much time and effort to get around it.  I fixed `css` issue and the issue that you commented out. Thanks for suggestions which are descriptive and show me the proper way.

**Comment by ArcoMul** [comment]
Super! @Serafettin99  👏
